### ISSUE TITLE
Reaction getUsers does not return all of the users

### ISSUE THREAD

**Issue by Tigerism** [SinisterRectus/Discordia#79]
```lua
p(reaction.count) --> correct number
p(#reaction:getUsers()) --> incorrect number
p(reaction:getUsers():count(function(r) return r end)) --> incorrect number
```
Since all of the users are not retrieved, iterating through them does not iterate through all of the users who reacted to the message.

**Comment by SinisterRectus** [comment]
Will fix ASAP. The issue is that this endpoint returns paginated results, but this was only recently documented. See discordapp/discord-api-docs#401.

**Comment by SinisterRectus** [comment]
Should be fixed by b9b181df3fa5f11b20f345f575fdc3ba295011c8.

**Comment by SinisterRectus** [comment]
If more than 100 reaction users exist. you'll need to make multiple requests to get them all. Until I fix cache sorting, you'll have to semi-manually sort the users:

```lua
local users = reaction:getUsers():toArray(id)
local moreUsers = reaction:getUsersAfter(users[#users])
```
